.vscode/launch.json

```diff
@@ -64,7 +64,7 @@
                 "pipeArgs": [
                     "exec",
                     "-i",
-                    "gateway-root",
+                    "gateway-${env:USER}",
```

Comment:
Note that this will not work if we run vscode as sudo, and then it needs to be `"gateway-${env:SUDO_USER}"`. But I don't know how to specify this in the launch.json file, so OK.

Reply:
Why would you run vscode as sudo?